src/app/src/components/FacilityListItems.jsx

```diff
@@ -146,6 +149,13 @@ class FacilityListItems extends Component {
                 </p>
             ) : null;
 
+        const awaitingModerationMessage = readOnly ? (
+            <p style={{ color: 'red' }}>
+                This list has matches awaiting moderation from the{' '}
+                {isAdminUser ? 'contributor' : 'OS Hub admins'}.
```

Comment:
@obrienad this is the new copy we wanted to run by OAR - I took a stab at it (you see this in the 3rd screenshot on the PR description as well).